js/piwik.js

```diff
@@ -5319,6 +5320,20 @@ if (typeof Piwik !== 'object') {
                 },
 
                 /**
+                 * Useful to debug content tracking. This method will log all detected content blocks to console
+                 * (if the browser supports the console). It will list the detected name, piece, and target of each
+                 * content block.
+                 */
+                logAllContenBlocksOnPage: function () {
```

Comment:
Thought about fixing this myself, but since the ticket description uses the apparent misspelling, I will wait for your input.